docs/how-to-guides/feature-flags.md

```diff
@@ -99,6 +102,6 @@ In this case, the minification process will remove the entire `if` statement inc
 
 ## Frequently asked questions
 
-### Why shouldn't I assign the result of an expression involving `IS_GUTENBERG_PLUGIN` to a variable, e.g. `const isMyFeatureActive = process.env.IS_GUTENBERG_PLUGIN === 2`?
+### Why shouldn't I assign the result of an expression involving `IS_GUTENBERG_PLUGIN` to a variable, e.g. `const isMyFeatureActive = globalThis.IS_GUTENBERG_PLUGIN === 2`?
```

Comment:
This needs to be updated, as it no longer fits. The original phrasing was related to the old constant that was a number. Now it's a boolean.

Reply:
```suggestion
### Why shouldn't I assign the result of an expression involving `IS_GUTENBERG_PLUGIN` to a variable, e.g. `const isMyFeatureActive = ! Object.is( undefined, globalThis.IS_GUTENBERG_PLUGIN )`?
```

Reply:
Actually, it probably doesn't matter. I see what this section tries to explain. Do not use it with complex expressions. Now, I'm wondering if tree-shaking would work properly with `const isActive = globalThis.IS_GUTENBERG_PLUGIN; if ( isActive ) { ... }`.